SevenZip/SevenZipCompressor.cs

```diff
@@ -215,10 +215,6 @@ private bool MethodIsValid(CompressionMethod method)
 
             switch (_archiveFormat)
             {
-                case OutArchiveFormat.Zip:
-                {
-                    return method != CompressionMethod.Ppmd;
```

Comment:
This didn't really make sense to me - PPMd is supported in both ZIP and 7z formats, so why this exclusion?